fabric-api-lookup-api-v1/src/main/java/net/fabricmc/fabric/api/lookup/v1/block/BlockApiCache.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.lookup.v1.block;
+
+import java.util.Objects;
+
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.block.entity.BlockEntity;
+import net.minecraft.world.World;
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.util.math.BlockPos;
+
+import net.fabricmc.fabric.impl.lookup.block.BlockApiCacheImpl;
+import net.fabricmc.fabric.impl.lookup.block.BlockApiLookupImpl;
+
+/**
+ * A {@link BlockApiLookup} bound to a {@link ServerWorld} and a position, providing much faster Api access.
+ * Refer to {@link BlockApiLookup} for example code.
+ *
+ * <p>{@link BlockApiLookup#get(World, BlockPos, BlockState, BlockEntity, Object) BlockApiLookup#get()} looks up
+ * the block state, the block entity, and the Api provider registered for the target block.
+ *
+ * <br>This object caches the block entity at the target position, and the last used Api provider, removing those queries.
```

Comment:
This is the only occurrence of `<br>`. Is it necessary for formatting or can it be replaced with a `<p>` for consistency?

Reply:
Thanks!